codechain/rpc/request.py

```diff
@@ -0,0 +1,14 @@
+from jsonrpcclient.clients.http_client import HTTPClient
+
+
+class RpcRequest:
+    def __init__(self, node: str):
+        self.client = HTTPClient(node)
+
+    def call(self, request):
+        response = self.client.send(request)
+
+        if not response.ok:
```

Comment:
I think they already do this: https://github.com/bcb/jsonrpcclient/blob/master/jsonrpcclient/client.py#L176
Then this `RpcRequest` class isn't necessary, I think.

Reply:
I removed it.